packages/next/src/client/components/react-dev-overlay/hot-reloader-client.tsx

```diff
@@ -481,7 +475,7 @@ export default function HotReload({
   const router = useRouter()
 
   useEffect(() => {
-    const handler = (event: MessageEvent<PongEvent>) => {
+    const handler = (event: MessageEvent<any>) => {
```

Comment:
We'll handle this in a follow-up PR, this was just removing the reference to `PongEvent`. This type was wrong anyway because `event` would be many other events too.